### PR TITLE
add windows build

### LINKED ISSUE #1: enable windows build
google-resumable-media requires google-crc32c (and thus this package) https://github.com/conda-forge/google-resumable-media-feedstock/pull/14, and I'd like to keep that Python package installable on Windows.

### PR DESCRIPTION
**Pull request by tswast**
Note: I don't have a Windows dev machine at the moment, so opening a PR to see what happens. Based bld.bat on https://github.com/conda-forge/libprotobuf-feedstock/blob/master/recipe/bld.bat and https://github.com/google/crc32c/blob/master/.appveyor.yml

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Closes #1

### CODE REVIEW

**Comment by tswast** [comment]
@conda-forge-admin, please rerender

**Comment by xhochy** [comment]
@tswast I've added some checks to ensure that all files are part of the package. Once concern here is that I don't see the import library `crc32c.lib` in the CMake output. Otherwise this looks good.

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+::https://github.com/google/crc32c/blob/master/.appveyor.yml
+cmake -G "Ninja" ^
+         -DBUILD_TESTING=OFF ^
+         -DBUILD_SHARED_LIBS=ON ^
+         -DCMAKE_BUILD_TYPE=Release ^
+         -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+         -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+         -DCRC32C_BUILD_TESTS=OFF ^
+         -DCRC32C_BUILD_BENCHMARKS=OFF ^
+         -DCRC32C_USE_GLOG=0 ^
```
Comment (xhochy) [inline recipe/bld.bat:16]: This might fix it.
```suggestion
         -DCRC32C_USE_GLOG=0 ^
         -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment (tswast) [reply recipe/bld.bat:16]: That worked! https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=187510&view=logs&j=f97ca392-d626-52d0-4b2e-4b27aa319a77&t=7b750183-140d-50ae-3ee1-cbef6218bdf6

Thanks for the help.

**Pull request merged**